models/migrations/20220727232515_init/migration.sql

```diff
@@ -0,0 +1,203 @@
+-- CreateEnum
```

Comment:
This file should be deleted as per the discussion on Discord: https://discord.com/channels/994239034362691765/995931261422010448/1006390907899228261

Reply:
I deleted the file. Thanks.